source/extensions/quic_listeners/quiche/platform/spdy_containers_impl.h

```diff
@@ -14,12 +20,7 @@
 // TODO(mpw): fix includes to use full paths.
 #include "spdy_string_impl.h"
```

Comment:
This TODO seems easy to fix, can you do it while we are here?

Reply:
done